frontend/src/app/modules/workspace/file-browser-section/file-browser-section.component.ts

```diff
@@ -92,7 +92,8 @@ export class FileBrowserSectionComponent implements OnInit {
             name: `New File ${++this.fileCounter}`,
             content: "// Start code here:\n",
             projectId: this.projectId,
-            folder : ""
+            folder : "",
+            creatorId: 1
```

Comment:
This is bad to hardcode creatorId. Can we remove it?

Reply:
This must be removed from every DTO not only frontend, somebody removes it partialy and this gives errors. This is just fix for  successful passing build check